eslint.config.mjs

```diff
@@ -3,37 +3,39 @@ import pluginJs from "@eslint/js";
 import tseslint from "typescript-eslint";
 import pluginReactConfig from "eslint-plugin-react/configs/recommended.js";
 
-
 export default [
-  pluginJs.configs.recommended,
-  ...tseslint.configs.recommended,
-  pluginReactConfig,
-  {
-    languageOptions: {
-      globals: {
-        ...globals.browser,
-        ...globals.jest,
-        browser: true,
-        module: true
-      }
+    {
+        ignores: [".next", "node_modules"]
     },
-    rules: {
-      indent: [
-          "error",
-          4
-        ],
-        "linebreak-style": [
-          "error",
-          "unix"
-        ],
-        quotes: [
-          "error",
-          "double"
-        ],
-        semi: [
-          "error",
-          "always"
-        ]
-    }
-  }
-];
\ No newline at end of file
+    pluginJs.configs.recommended,
+    ...tseslint.configs.recommended,
+    pluginReactConfig,
+    {
+        languageOptions: {
+            globals: {
+                ...globals.browser,
+                ...globals.jest,
+                ...globals.node,
+                browser: true,
+                module: true
+            }
+        },
+        rules: {
+            "react/react-in-jsx-scope": "off",
+            "react/no-unescaped-entities": "off",
+            "no-unused-vars": "off",
+            "@typescript-eslint/no-unused-vars": "off",
```

Comment:
I think I would like to recommend we don't turn this off, but instead figure out how to use the [argsIgnorePattern](https://eslint.org/docs/latest/rules/no-unused-vars#argsignorepattern) because that way we can be really clear when we don't want it to check, but it will still check by default